_includes/js/users.md

```diff
@@ -362,68 +362,60 @@ Our library manages the `FB` object for you. The `FB` singleton is synchronized
+Parse allows you to link your users with [3rd party authentication ]({{ site.baseUrl }}/parse-server/guide/#oauth-and-3rd-party-authentication), enabling your users to sign up or log into your application using their existing identities. This is accomplished through `_linkWith` method by providing authentication data for the service you wish to link to a user in the `authData` field. Once your user is associated with a service, the `authData` for the service will be stored with the user and is retrievable by logging in.
 
-`authData` is a JSON object with keys for each linked service containing the data below. In each case, you are responsible for completing the authentication flow (e.g. OAuth 1.0a using hellojs on client side) to obtain the information the the service requires for linking.
+`authData` is a JSON object with keys for each linked service containing the data below. In each case, you are responsible for completing the authentication flow by creating your own Authentication Provider.
 
-### Facebook `authData`
+### Authentication Provider
 
-```javascript
-{
-  "facebook": {
-    "id": "user's Facebook id number as a string",
-    "access_token": "an authorized Facebook access token for the user",
-    "expiration_date": "token expiration date of the format: yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
-  }
-}
-```
-Learn more about [Facebook login](https://developers.facebook.com/docs/authentication/).
+To link your users to a [3rd party authentication]({{ site.baseUrl }}/parse-server/guide/#oauth-and-3rd-party-authentication), you must create a Authentication Provider. Similar to `Parse.Object` subclass, you must register with `Parse.User._registerAuthenticationProvider`.
 
-### Twitter `authData`
+Here is a minimal example to link with [Twitter Auth](http://docs.parseplatform.org/parse-server/guide/#twitter-authdata):
 
 ```javascript
-{
-  "twitter": {
-    "id": "user's Twitter id number as a string",
-    "screen_name": "user's Twitter screen name",
-    "consumer_key": "your application's consumer key",
-    "consumer_secret": "your application's consumer secret",
-    "auth_token": "an authorized Twitter token for the user with your application",
-    "auth_token_secret": "the secret associated with the auth_token"
-  }
-}
-```
-
-Learn more about [Twitter login](https://dev.twitter.com/docs/auth/implementing-sign-twitter).
+const authData = {
+  "id": "12345678",
+  "screen_name": "ParseIt",
+  "consumer_key": "SaMpLeId3X7eLjjLgWEw",
+  "consumer_secret": "SaMpLew55QbMR0vTdtOACfPXa5UdO2THX1JrxZ9s3c",
```

Comment:
what credentials are these?

Reply:
Fake credentials. This code doesn’t work because twitter docs need to be updated too. Can you provide feedback https://github.com/parse-community/Parse-SDK-JS/issues/771#issuecomment-486735957